### ISSUE TITLE
Usage Logging crashing when calling Create.

### ISSUE THREAD

**Issue by BenjaminGibbs** [recurly/recurly-client-net#168]
So I have ran in to a problem when logging usage. The exception is "{"Input string was not in a correct format."}"

The usage gets logged in the portal just fine but it appears the return message is not getting serialized correctly.

```
   at System.Number.ParseSingle(String value, NumberStyles options, NumberFormatInfo numfmt)
   at System.Xml.XmlConvert.ToSingle(String s)
   at System.Xml.XmlReader.ReadElementContentAsFloat()
   at Recurly.Usage.ReadXml(XmlTextReader reader) in C:\Users\ben\Documents\GitHubVisualStudio\recurly-client-net\Library\Usage.cs:line 114
   at Recurly.Client.ReadWebResponse(HttpWebResponse response, ReadXmlDelegate readXmlDelegate, ReadXmlListDelegate readXmlListDelegate, ReadResponseDelegate responseDelegate) in C:\Users\ben\Documents\GitHubVisualStudio\recurly-client-net\Library\Client.cs:line 382
   at Recurly.Client.PerformRequest(HttpRequestMethod method, String urlPath, WriteXmlDelegate writeXmlDelegate, ReadXmlDelegate readXmlDelegate, ReadXmlListDelegate readXmlListDelegate, ReadResponseDelegate reseponseDelegate) in C:\Users\ben\Documents\GitHubVisualStudio\recurly-client-net\Library\Client.cs:line 170
   at Recurly.Usage.Create() in C:\Users\ben\Documents\GitHubVisualStudio\recurly-client-net\Library\Usage.cs:line 56
   at recurlyupdatedtest.MainWindow.GetAccount() in C:\Users\benja\Documents\Visual Studio 2015\Projects\recurlyupdatedtest\recurlyupdatedtest\MainWindow.xaml.vb:line 63
   at recurlyupdatedtest.MainWindow.VB$StateMachine_1_button_Click.MoveNext() in C:\Users\benja\Documents\Visual Studio 2015\Projects\recurlyupdatedtest\recurlyupdatedtest\MainWindow.xaml.vb:line 8
```

The code I am using to get this is:

```vbnet
        Dim demoAccount = Accounts.Get(accountKey)
        Dim subscriptions = demoAccount.GetSubscriptions(Subscription.SubscriptionState.Active)
        Dim firstSub = subscriptions(0)
        Dim usage = New Usage(firstSub.Uuid, "upload")
        usage.Amount = 1
        usage.MerchantTag = "File Uploaded by xxxxx"
        usage.UsageTimestamp = Date.UtcNow.AddMinutes(-5)
        usage.Create()
```

The XML I am getting back from the server is:

```xml
<usage href="https://xxx.recurly.com/v2/subscriptions/38b94a368c8548d26a3f0249da8fe29b/add_ons/upload/usage/500057084407056110">
  <measured_unit href="https://xxx.recurly.com/v2/measured_units/480236656838837542"/>
  <amount type="integer">1</amount>
  <merchant_tag>File Uploaded by xxxxx</merchant_tag>
  <recording_timestamp type="date">2016-09-21T02:39:33+00:00</recording_timestamp>
  <usage_timestamp type="date">2016-09-21T02:39:33+00:00</usage_timestamp>
  <created_at type="date">2016-09-21T02:44:09+00:00</created_at>
  <updated_at type="date">2016-09-21T02:44:09+00:00</updated_at>
  <billed_at nil="nil"></billed_at>
  <usage_type>price</usage_type>
  <unit_amount_in_cents type="integer">1</unit_amount_in_cents>
  <usage_percentage nil="nil"></usage_percentage>
</usage>
```

So I am thinking that `reader.ReadElementContentAsFloat();` is crashing when it runs because `<usage_percentage nil="nil"></usage_percentage>` has no value?

**Comment by bhelx** [comment]
@BenjaminGibbs I'll try to reproduce this now. Will report back

**Comment by bhelx** [comment]
@BenjaminGibbs Can you confirm this works for you? https://github.com/recurly/recurly-client-net/pull/169

**Comment by BenjaminGibbs** [comment]
It looks like it should, let me check it out.

**Comment by BenjaminGibbs** [comment]
This fix does work for me.

**Comment by bhelx** [comment]
Thanks, I'll get out a patch release and report back.

**Comment by bhelx** [comment]
Released in 1.4.1 https://github.com/recurly/recurly-client-net/releases/tag/1.4.1
